packages/spright-components/src/chat/message/index.ts

```diff
@@ -34,7 +34,7 @@ export class ChatMessage extends FoundationElement {
      * HTML Attribute: message-type
      */
     @attr({ attribute: 'message-type' })
-    public readonly messageType: ChatMessageType = ChatMessageType.system;
+    public messageType: ChatMessageType = ChatMessageType.system;
```

Comment:
This was just an oversight. It's necessary to be able to change a message type after construction, for example in `chat-conversation.stories.ts` where we construct a message from TS code when the user presses Send.